src/language/parser.ts

```diff
@@ -91,23 +91,6 @@ export interface ParseOptions {
    */
   maxTokens?: number | undefined;
 
-  /**
-   * @deprecated will be removed in the v17.0.0
-   *
-   * If enabled, the parser will understand and parse variable definitions
-   * contained in a fragment definition. They'll be represented in the
-   * `variableDefinitions` field of the FragmentDefinitionNode.
-   *
-   * The syntax is identical to normal, query-defined variables. For example:
-   *
-   * ```graphql
-   * fragment A($var: Boolean = false) on T {
-   *   ...
-   * }
-   * ```
-   */
-  allowLegacyFragmentVariables?: boolean | undefined;
```

Comment:
Will update this to `enableExperimentalFragmentArguments?: boolean | undefined` and use this so we can turn on fragment arguments within *tooling* prior to *servers* supporting them, which should unblock *Relay* fully supporting the syntax (even in open source) and using compiler transforms to remap fragments with arguments to duplicated fragment definitions.

Then we'll be able to add a flag to Relay's compiler on whether the *server* supports fragment spreads, and then when we finally merge the spec we'll be able to remove the flags entirely.